ligo/types.mligo

```diff
@@ -372,7 +372,9 @@ type y_to_x_rec_param = x_to_y_rec_param
 
 type oracle_view_param = cumulatives_value list
 
-type observe_param = {
+type observe_param =
+[@layout:comb]
```

Comment:
Not that I'm particularly against it, but why the change in layout?

Reply:
Without it, LIGO will reorder fields, and this will not comply with the TZIP View pattern. I supposed it is better to comply with it rather than not )